### PR TITLE
{math}[foss/2019b] polymake v4.0r1

### PR DESCRIPTION
**Pull request by terjekv**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by terjekv** [comment]
Depends on #9901.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2548.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1f43d3696ae249fd0f8b989e7af451c7 for a full test report.

File: easybuild/easyconfigs/p/polymake/polymake-4.0r1-foss-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    ('libreadline', '8.0'),
+    ('Perl', '5.30.0'),
+    ('SVG', '2.84', '-Perl-5.30.0'),
+    ('GMP', '6.1.2'),
+    ('MPFR', '4.0.2'),
+    ('Boost', '1.71.0'),
+    ('Java', '1.8', '', True),
+    ]
+
+# configopts = "--enable-libnuma=$EBROOTNUMACTL"
```
Comment (boegel) [inline easybuild/easyconfigs/p/polymake/polymake-4.0r1-foss-2019b.eb]: @terjekv This commented out line can be removed?

Also, please add a custom `sanity_check_paths`. Here's a suggestion:

```python
sanity_check_paths = {
    'files': ['bin/polymake', 'bin/polymake-config', 'lib/libpolymake.%s' % SHLIB_EXT],
    'dirs': ['include/polymake', 'lib/polymake', 'share/polymake'],
}
```
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/e602991fca5843019b0cde162e6450ca for a full test report.

**Comment by terjekv** [comment]
Test report by @terjekv
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in this PR)
ninhursaga.uio.no - Linux RHEL 8.1, Intel(R) Xeon(R) Silver 4216 CPU @ 2.10GHz, Python 3.6.8
See https://gist.github.com/80d0dcb6814712d311a18c8ca4ad08ed for a full test report.

File: easybuild/easyconfigs/p/polymake/polymake-4.0r1-foss-2019b.eb (hunk 1 above)
Comment (terjekv) [reply easybuild/easyconfigs/p/polymake/polymake-4.0r1-foss-2019b.eb]: Sorted in https://github.com/easybuilders/easybuild-easyconfigs/pull/9904/commits/bcea366517b89bdf5c7518754ba87b5851ec7371.

**Comment by terjekv** [comment]
As for the removal of Java... https://polymake.org/doku.php/install/install?s[]=java states that:

> Java Development Kit 1.5+ [is required] if you are going to use the visualization tools JavaView or jReality via polymake.

Neither JavaView or jReality are installed with this build, and as such, Java isn't needed.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3101.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/eb00521f4814eaf4bb43cf9734997d15 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2588.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d8d882533075978800bcec8099a66f7b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/e746334c9b395f52ff8610113be0da50 for a full test report.

**Comment by terjekv** [comment]
> Test report by @boegel
> **FAILED**
> Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
> node2588.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
> See https://gist.github.com/d8d882533075978800bcec8099a66f7b for a full test report.

Bus error? :(

**Comment by boegel** [comment]
> > Test report by @boegel
> > **FAILED**
> > Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
> > node2588.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
> > See https://gist.github.com/d8d882533075978800bcec8099a66f7b for a full test report.
> 
> Bus error? :(

@terjekv That test was run on a partial node (1/4th), and apparently ~16GB of RAM is not sufficient to build `polymake` from source using 6 cores...

Will submit another test from a full node.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2557.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/11198a23c1279201a5df83629c9d8e00 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @terjekv!

**Pull request merged**